code/src/main.tsx

```diff
@@ -2,7 +2,9 @@ import { Provider } from "react-redux";
 import { createRoot } from "react-dom/client";
 import { RouterProvider, createBrowserRouter } from "react-router-dom";
 import { StyledEngineProvider } from "@mui/material";
+
 import I18nProivider from "@/context/I18nProvider";
+import { ModalProvider } from "@/context/ModalContext";
```

Comment:
the same here. Don't forget to follow imports structure and also spaces
The space is only for `styles` in file